roles/elastic-stack/ansible-elasticsearch/tasks/Debian.yml

```diff
@@ -30,13 +30,13 @@
 
 - name: Debian/Ubuntu | Add Elasticsearch GPG key.
   apt_key:
-    url: "https://artifacts.elastic.co/GPG-KEY-elasticsearch"
+    url: "{{ elasticrepo.gpg }}"
+    id: 46095ACC8548582C1A2699A9D27D666CD88E42B4
```

Comment:
Parameter `id` should be used with a variable declared in ` roles/elastic-stack/ansible-elasticsearch/defaults/main.yml ` that would ease the configuration and update in case the GPG key expires or needs to be updated.

Please review occurrences and other `id` values like the Wazuh ones.

Reply:
Thx for the suggestion. I added the parameter and adjusted all occurrences.